kuyruk/task.py

```diff
@@ -194,6 +198,9 @@ def _module_name(self):
         return name
 
 
+SubTask = namedtuple("SubTask", "task, args, kwargs, host")
```

Comment:
I would use a tuple instead of a string:

```python
SubTask = namedtuple("SubTask", ("task", "args", "kwargs", "host"))
```

Is there any benefit of having them in a comma-separated string instead of a tuple or a list?